proposals/stage-1/CODE_OF_CONDUCT/HANDLING-REPORTS.md

```diff
@@ -0,0 +1,88 @@
+# Handling reports and escalation
+This document describes the proposed process for handling reports and escalation, along with relevant roles in the OpenJS Foundation.
+
+This process covers two types of reports based on ./FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md:
+
+* Reports for spaces managed by the Cross Project Council (CPC) which come in through `report@opensjsf.org`
+* Escalations which come in through `coc-escalation@openjsf.org`.   
+
+Generally, reports to coc-escalation should refer to a report that was already submitted to:
+
+* report@openjsf.org or
+* one of the projects reporting emails 
```

Comment:
projects is meant to be possessive as in `projects'` here, correct?

Reply:
Fixed